BreakOut/spinn_breakout/python_models/breakout_machine_vertex.py

```diff
@@ -13,21 +13,19 @@
 # ----------------------------------------------------------------------------
 # BreakoutMachineVertex
 # ----------------------------------------------------------------------------
-class BreakoutMachineVertex(MachineVertex,
-                            ProvidesProvenanceDataFromMachineImpl):
+class BreakoutMachineVertex(MachineVertex):
 
     _BREAKOUT_REGIONS = Enum(
         value="_BREAKOUT_REGIONS",
         names=[('SYSTEM', 0),
-               ('BREAKOUT', 1),
-               ('PROVENANCE', 2)])
+               ('BREAKOUT', 1)])
```

Comment:
I can't entirely remember how this worked in the C code but shouldn't that be changed to match?